scripts/jobs/customer_services/vonage_one_time_ingestion.py

```diff
@@ -45,10 +45,7 @@ def get_auth_token(client_id, client_secret, scope):
     print(f"Auth Token status Code = {r.status_code}")
     auth_token = json.loads(r.text)
 
-    if(r.status_code == 200):
-        return auth_token
-    else:
-        r.raise_for_status()
+    return auth_token
```

Comment:
is there a reason you're removing this status code check?

Reply:
Its gone in my branch, I think we should put it back in though.